templates/repo/commits_list_small.tmpl

```diff
@@ -16,8 +16,21 @@
 
 		<span class="ui float right shabox">
 			{{if eq (CommitType .) "SignCommitWithStatuses"}}
-        		{{template "repo/commit_status" .Status}}
-        	{{end}}
+				<div class="ui dropdown">
+					<span class="commit-status">{{template "repo/commit_status" .Status}}</span>
+					<div class="menu commit-statuses">
+    				{{range .Statuses}}
+        			<div class="ui item">
+            		<span>{{template "repo/commit_status" .}}</span>
+            		<span class="ui">{{.Context}} <span class="text grey">{{.Description}}</span></span>
+            		<div class="ui right">
+									<span class="ui">{{if .TargetURL}}<a href="{{.TargetURL}}">Details</a>{{end}}</span>
```

Comment:
I think those `rel` attributes still have some security implications so would add them back. Unfortunately I don't think we can factor out these attributes into a helper or subtemplate because one cannot pass inner HTML to those that would be needed for link tags.

Reply:
The current state of dropdown is very much just something to show that it works, I'm unsure how to style it further to be honest using Fomantic's dropdown; will most likely have to create our own element for this since I don't see much in a way of styling either `item` or `message` types.

Possibly popup element can be used instead, I'll have to experiment.

Reply:
Regarding popovers and tooltips, I was thinking about integrating tippy.js. We could create our own HTML scheme to make it work, something like:

```html
<div class="dropdown" data-delay="100">
  regular HTML
  <div class="dropdown-content">
    dropdown HTML
  </div>
</div>
```

`.dropdown-content` would be invisible on page load and be initialized via JS. Should probably use the "headless" variant that comes without any CSS and style it on our own.

https://atomiks.github.io/tippyjs/#html-content
https://atomiks.github.io/tippyjs/v6/html-content/
https://atomiks.github.io/tippyjs/v6/headless-tippy/